### PR TITLE
Fix mediawiki version

### PR DESCRIPTION
**Pull request by Hyper-Node**
# MaRDI Pull Request

**Changes**:
- fixing mediawiki version so no unforeseen increments happen overnight (asynchronous mediawiki and extension which happens if the latest version increments) 

**Instructions for PR review**:
- [x] Conceptual Review (Logic etc...) 
- [ ] Code Review (Review your implementation) 
- [ ] Checkout (Test changes locally) 

**Checklist for this PR**: 
- [x] [Reviewers and Assignee specified.](https://docs.github.com/en/issues/tracking-your-work-with-issues/assigning-issues-and-pull-requests-to-other-github-users)
- [ ] [All related issues are linked.](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)

### CODE REVIEW

**Comment by Hyper-Node** [comment]
https://github.com/MaRDI4NFDI/portal-compose/issues/249

**Review by physikerwelt — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -8,12 +14,12 @@ RUN apt-get update && \
     apt-get install --reinstall ca-certificates && \
     apt-get clean && rm -rf /var/lib/apt/lists/*
 
+# fetcher specific settings
 # clone extensions from github, using specific branch
-
 ARG WMF_BRANCH=wmf/1.39.0-wmf.13
```
[the remark this answers is not in the document]
Comment (Hyper-Node) [reply Dockerfile:19]: Tested this, it doesn't resolve the arguments then. 
If putting the ARG for MEDIAWIKI_VERSION below, then its the same result for this statement.

[the remark this answers is not in the document]
Comment (physikerwelt) [reply Dockerfile:19]: Strange. So if you move all args to the top they don't work. This is hard to explain

[the remark this answers is not in the document]
Comment (dajuno) [reply Dockerfile:19]: https://docs.docker.com/engine/reference/builder/#scope
https://stackoverflow.com/a/53682110/4817051

[the remark this answers is not in the document]
Comment (dajuno) [reply Dockerfile:19]: ah sorry, in this specific case it's exactly the opposite: https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact
- ARG used in FROM need to be defined before the first FROM
- ARG to be used as env vars inside FROM are valid only until the next FROM

[the remark this answers is not in the document]
Comment (dajuno) [reply Dockerfile:19]: no. only args used in FROM statements can be moved to the top. the others are local for the respective build stage where they are defined.

[the remark this answers is not in the document]
Comment (dajuno) [reply Dockerfile:19]: well turns out it is possible like this: https://github.com/MaRDI4NFDI/docker-wikibase/pull/47

**Pull request merged**